builtin/fsmonitor--daemon.c

```diff
@@ -1305,9 +1305,11 @@ static int try_to_run_foreground_daemon(int free_console)
 		die("fsmonitor--daemon is already running '%s'",
 		    the_repository->worktree);
 
-	printf(_("running fsmonitor-daemon in '%s'\n"),
-	       the_repository->worktree);
-	fflush(stdout);
+	if (!quiet) {
+		printf(_("running fsmonitor-daemon in '%s'\n"),
```

Comment:
Please do print to `stderr`. This output is not intended to be piped into any other process as `stdin`, but it is a status message much like a progress meter.